Contents/mods/CommunityAPI/media/lua/client/ExpandedModSelector/ModSelector.lua

```diff
@@ -0,0 +1,890 @@
+require("OptionsScreens/ModSelector")
+
+local FONT_HGT_TITLE = getTextManager():getFontHeight(UIFont.Title)
+local FONT_HGT_SMALL = getTextManager():getFontHeight(UIFont.Small)
+local FONT_HGT_MEDIUM = getTextManager():getFontHeight(UIFont.Medium)
+local FONT_HGT_LARGE = getTextManager():getFontHeight(UIFont.Large)
+
+ModSelector = ISPanelJoypad:derive("ModSelector");
```

Comment:
We must stop using globals as much as possible.
If possible, set to local and return at the end of file.
Require in the files that needs it.

Reply:
Done